metpy/calc/thermo.py

```diff
@@ -354,6 +354,9 @@ def lfc(pressure, temperature, dewpt, parcel_temperature_profile=None):
             x, y = lcl(pressure[0], temperature[0], dewpt[0])
             return x, y
     else:
+        idx = x < lcl(pressure[0], temperature[0], dewpt[0])[0]
```

Comment:
A comment for what this else block is doing would be helpful.

Reply:
Fair